.github/CODEOWNERS

```
@@ -1,110 +1,73 @@
-
-# FPGA extensions
-sycl/include/sycl/ext/intel/fpga_device_selector.hpp @MrSidims
-sycl/include/sycl/ext/intel/fpga_extensions.hpp @MrSidims
-sycl/include/sycl/ext/intel/fpga_reg.hpp @MrSidims
-sycl/include/sycl/ext/intel/pipes.hpp @MrSidims
-sycl/include/CL/sycl/pipes.hpp @MrSidims
-
-# Reduction extension
-sycl/include/sycl/ext/intel/reduction.hpp @v-klochkov
-
-# Function pointers
-sycl/include/sycl/ext/intel/function_pointer.hpp @AlexeySachkov
-sycl/source/function_pointer.cpp @AlexeySachkov
-
-# Half Type
-sycl/include/CL/sycl/half_type.hpp @AlexeySachkov
-sycl/source/half_type.cpp @AlexeySachkov
-
 # XPTI instrumentation utilities
-xpti/ @tovinkere @andykaylor @alexbatashev
-xptifw/ @tovinkere @andykaylor @alexbatashev
+xpti/ @tovinkere @intel/llvm-reviewers-runtime
+xptifw/ @tovinkere @intel/llvm-reviewers-runtime
 
 # DPC++ tools
-llvm/**/append-file/ @mdtoguchi @AGindinson
-llvm/**/file-table-tform/ @mlychkov @AlexeySachkov @kbobrovs
-llvm/**/llvm-foreach/ @AlexeySachkov  @Fznamznon
-llvm/**/llvm-no-spir-kernel/ @AGindinson @AlexeySachkov
-llvm/**/sycl-post-link/ @mlychkov @AlexeySachkov @kbobrovs
-llvm/include/llvm/Support/PropertySetIO.h @mlychkov @AlexeySachkov @kbobrovs
-llvm/lib/Support/PropertySetIO.cpp @mlychkov @AlexeySachkov @kbobrovs
-llvm/unittests/Support/PropertySetIOTest.cpp @mlychkov @AlexeySachkov @kbobrovs
-llvm/include/llvm/Support/SimpleTable.h @mlychkov @AlexeySachkov @kbobrovs
-llvm/lib/Support/SimpleTable.cpp @mlychkov @AlexeySachkov @kbobrovs
-llvm/unittests/Support/SimpleTableTest.cpp @mlychkov @AlexeySachkov @kbobrovs
-llvm/lib/Support/Base64.cpp @mlychkov @AlexeySachkov @kbobrovs
+llvm/**/append-file/ @intel/dpcpp-tools-reviewers
+llvm/**/file-table-tform/ @intel/dpcpp-tools-reviewers
+llvm/**/llvm-foreach/ @intel/dpcpp-tools-reviewers
+llvm/**/llvm-no-spir-kernel/ @intel/dpcpp-tools-reviewers
+llvm/**/sycl-post-link/ @intel/dpcpp-tools-reviewers
+llvm/include/llvm/Support/PropertySetIO.h @intel/dpcpp-tools-reviewers
+llvm/lib/Support/PropertySetIO.cpp @intel/dpcpp-tools-reviewers
+llvm/unittests/Support/PropertySetIOTest.cpp @intel/dpcpp-tools-reviewers
+llvm/include/llvm/Support/SimpleTable.h @intel/dpcpp-tools-reviewers
+llvm/lib/Support/SimpleTable.cpp @intel/dpcpp-tools-reviewers
+llvm/unittests/Support/SimpleTableTest.cpp @intel/dpcpp-tools-reviewers
+llvm/lib/Support/Base64.cpp @intel/dpcpp-tools-reviewers
 
 # Clang offload tools
-clang/tools/clang-offload-bundler/ @mlychkov @sndmitriev @AlexeySachkov
-clang/tools/clang-offload-wrapper/ @mlychkov @sndmitriev @AlexeySachkov @kbobrovs
-clang/tools/clang-offload-deps/ @sndmitriev @mlychkov @AlexeySachkov
-clang/tools/clang-offload-extract/ @sndmitriev @mlychkov @AlexeySachkov
+clang/tools/clang-offload-bundler/ @intel/dpcpp-tools-reviewers
+clang/tools/clang-offload-wrapper/ @intel/dpcpp-tools-reviewers
+clang/tools/clang-offload-deps/ @intel/dpcpp-tools-reviewers
+clang/tools/clang-offload-extract/ @intel/dpcpp-tools-reviewers
```

```suggestion
clang/tools/clang-offload-*/ @intel/dpcpp-tools-reviewers
```
How about matching all clang tools named with `clang-offload-` prefix.